sources/packages/web/src/composables/useEmitter.ts

```diff
@@ -0,0 +1,6 @@
+import mitt from "mitt";
+const emitter = mitt();
+
+export default function useEmitter() {
```

Comment:
👍

I assume they are event emit and listen at global or root level. 
One thing to be noted here is global event emitters/listeners must be used minimally where there is no other way to achieve the communication, otherwise it is not encouraged.